test/13_subdir_package/bin/before_build.py

```diff
@@ -0,0 +1 @@
+print('before_build.py executed!')
```

Comment:
I do not see any check for this.

Reply:
I don't check that it ran, that's covered by a different test. The test just verifies that it's looking up the correct path - if it didn't find the file at `{project}/bin/before_build.py`, we'd have a failed test.

As an aside, how would I test for this? It would be nice to be able to assert inside tests based on the stdout of cibuildwheel. I was thinking we could return a named tuple from run_cibuildwheel, but would pytest `capfd` work perhaps?

Reply:
I prefer to use fixture `capsys` as in `main_patform_test.py`

Reply:
would that capture the output of a subprocess?

Reply:
From my experience yes.

Reply:
If it's easy enough, it can't hurt to check it again?

Reply:
I've added these asserts using `capfd` - `capsys` didn't capture anything FYI.

Reply:
Somehow the output of the subcommand would need to be forwarded to `sys.stdout`/`sys.stderr` to have `capsys` work, I guess?
But that's apparently non-trivial to do: https://stackoverflow.com/questions/18421757/live-output-from-subprocess-command. Or you'd need to have the `subprocess.check_call` replaced by `subprocess.run` and return the output, which would also not be so clean anymore, since now the list of wheels is returned.
So hurray for `capfd`! :-)